{{ cookiecutter.repo_name }}/OWNERS

```diff
@@ -1,8 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - durandom
-  - tumido
+{%- set applist = cookiecutter.approvers.split(',') %}
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 4: found character that cannot start any token.

Reply:
Should we ignore this comment @chauhankaranraj ?

Also, I don't think it makes sense to include durandom or tumido as default for all repos

Reply:
> Should we ignore this comment @chauhankaranraj ?

Yeah I think it can be ignored, coz the actual OWNERS file created from this template won't have `{{ ... }}` and all the templating stuff.

> Also, I don't think it makes sense to include durandom or tumido as default for all repos

Cool, I can remove these from the default approver/reviewer lists :+1:

Reply:
OWNERS file